config/kcp/apiexport_release.yaml

```diff
@@ -6,6 +6,25 @@ kind: APIExport
 metadata:
   name: release
 spec:
+  permissionClaims:
```

Comment:
@davidmogar am I missing any CRDs we rely on? I don't have dummy CRs to do an end to end flow since the binding changes so I am not sure. Maybe we should get together and run through the flow e2e in kcp together

Reply:
Not that I'm aware of. We will need GitOpsDeployments, but not yet. Regarding the getting together, it sounds good.